### PR TITLE
Update TAXES_IN_NORWAY.md

### PR DESCRIPTION
**Pull request by onmyway133**
- Update image link
- Fix typo

### CODE REVIEW

File: TAXES_IN_NORWAY.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ money from your last employer the first year.
 
 ### VAT (MVA or moms in norwegian)
 
-So this is the annoying tax that applies to most gods and services that
+So this is the annoying tax that applies to most goods and services that
```
Comment (sindrenm) [inline TAXES_IN_NORWAY.md]: Haha, LOL. Nice find, thanks. 😃

File: TAXES_IN_NORWAY.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ house and live on the street by the first sign of trouble. Yay!
 
 Happy programmer:
 
-![Happy programmer](http://www.rdegges.com/images/2012/programming.gif)
```
Comment (sindrenm) [inline TAXES_IN_NORWAY.md]: Huh, this link is dead, eh? Thanks!

File: TAXES_IN_NORWAY.md (hunk 1 above)
Comment (fespinoza) [inline TAXES_IN_NORWAY.md]: there is no tax applicable to gods here in norway

Thor lives tax free :D

**Comment by sindrenm** [comment]
Thanks for the fixes, @onmyway133! :smile:

**Pull request merged**